src/core/nixl_enum_strings.cpp

```
@@ -0,0 +1,85 @@
+    static const std::array strings = {
+        "READ", "WRITE"
+    };
+
+    if ((op < 0) || (op >= strings.size())) {
+        return "BAD_OP";
+    }
+    return strings[op];
+}
+
+std::string
+statusStr(const nixl_status_t &status) {
```

Cppcheck (2.20.0)

[style] The function 'statusStr' is never used.

(unusedFunction)